src/themes/components/carousel.ts

```diff
@@ -155,6 +228,17 @@ const Carousel: ComponentStyleConfig = {
         backgroundColor: 'gray.900',
       },
     },
+    dotsPaginationContainer: {
+      position: 'absolute',
+      width: 'full',
+      bottom: '5%',
```

Comment:
I see in the design it's fixed to 16px. Why did you go with percentage? This will change based on the slide size?

Reply:
forgot to revert it back